project1/applications.py

```diff
@@ -47,6 +49,38 @@ def result():
     else:
         return render_template("Registration.html")
 
+@app.route("/bookpage", methods=[ 'POST', 'GET'])
+def review_page():
```

Comment:
This page should not be bypassed. So please check with the session variable and redirect to home page if bypassed.